utils/rpms/daos.spec

```diff
@@ -228,6 +228,9 @@ Requires: lbzip2
 Requires: attr
 %if (0%{?suse_version} >= 1315)
 Requires: libpsm_infinipath1
+Requires: lua-lmod
+%else
+Requires: Lmod
```

Comment:
Could you please advise how this is related to this patch?

Reply:
It's a pre-existing bug being fixed.  Since our functional test framework uses `module {install,avail,etc}` we need to have the _modules_ packages on the test nodes.  This is an unfortunate side-effect of pre-installing too much onto the test node images.  It papers over missing dependencies in the RPMs.

As much as we want to pre-install as much as we can into the test images for efficiency, we do so at the risk of letting bugs like this go unnoticed.  I only noticed it when I was testing on my own local VM cluster, which I don't build with our test images but build with Vagrant and a vanilla OS image.